float32 grasp_x_px
float32 grasp_y_px
float32 grasp_theta


